packages/tx/src/features/util.ts

```diff
@@ -0,0 +1,138 @@
+import { Common, Mainnet } from '@ethereumjs/common'
+import {
+  Address,
+  MAX_INTEGER,
+  MAX_UINT64,
+  bigIntToHex,
+  bytesToBigInt,
+  bytesToHex,
+  toBytes,
+} from '@ethereumjs/util'
+
+import { paramsTx } from '../params.js'
+import { checkMaxInitCodeSize, validateNotArray } from '../util.js'
+
+import type { TransactionInterface, TransactionType, TxData, TxOptions } from '../types.js'
+
+export function getCommon(common?: Common): Common {
+  return common?.copy() ?? new Common({ chain: Mainnet })
+}
+
+/**
+ * Validates that an object with BigInt values cannot exceed the specified bit limit.
+ * @param values Object containing string keys and BigInt values
+ * @param bits Number of bits to check (64 or 256)
+ * @param cannotEqual Pass true if the number also cannot equal one less the maximum value
+ */
+export function valueBoundaryCheck( // TODO: better method name
```

Comment:
For this and the other util methods:
In the context of composition rather than inheritance, I feel like we should be a bit more explicit with the naming (e.g. with reference to "sharedTxConstructor" rather than "shared constructor"). I'm not sure if this method (and the other method utils) are intended to be used outside of a transaction context, but if they are exported outside of the package, I think we have be particularly explicit with naming. This is actually one of the benefits of class-based methods imo (not that I don't prefer this pattern), is they are naturally namespaced by virtue of the class they belong to.

Reply:
These are not supposed to be exported outside of the package! These are only supposed to be used internally. These "shared" methods nevertheless have to be named differently. We also have the opportunity now (due to breaking releases) to give everything a more clear name!